build.gradle

```diff
@@ -238,6 +238,7 @@ project('client') {
         compile project(":shared:controller-api")
         compile group: 'io.netty', name: 'netty-transport-native-epoll', version: nettyVersion
         compile group: 'com.google.guava', name: 'guava', version: guavaVersion
+        compile group: 'com.google.code.gson', name: 'gson', version: gsonVersion
```

Comment:
You're adding a new dependency to the client. Is this absolutely necessary? How big is the JAR? I would prefer not to make the client bigger than it currently is.

Reply:
I've modified the code now to avoid the dependency for Gson. I now use regex to extract the expiry time from the JWT token. Gson is now used for easier unit testing only. I hope that works.